### PR TITLE
Fixes #4748 - Increment "attachmentcount" in the "_threads" table

### LINKED ISSUE #4748: "attachmentcount" in the "_threads" table is not incremented
When the attachment is approved in the Mod-CP, the "attachmentcount" in the "_threads" table is not incremented. If there are no other attachments in the respective thread yet, the counter remains at 0 and thus the attachment is not displayed to users without mod permission.

https://community.mybb.com/thread-239024.html

### PR DESCRIPTION
**Pull request by Eldenroot**
Resolves #4748

### CODE REVIEW

**Comment by Eldenroot** [comment]
Typo fixed, srry :)

File: modcp.php [hunk 1]
```diff
@@ -2064,32 +2064,40 @@
 	{
 		$attachments = array_map("intval", array_keys($mybb->input['attachments']));
 		$query = $db->query("
-			SELECT a.pid, a.aid
+			SELECT a.pid, a.aid, t.tid
 			FROM  ".TABLE_PREFIX."attachments a
 			LEFT JOIN ".TABLE_PREFIX."posts p ON (a.pid=p.pid)
 			LEFT JOIN ".TABLE_PREFIX."threads t ON (t.tid=p.tid)
 			WHERE aid IN (".implode(",", $attachments)."){$tflist_queue_attach}
-		");
```
Comment (dvz) [inline modcp.php]: The indentation for this section got mistakenly changed (see whitespace differences)

**Review by dvz — CHANGES_REQUESTED** [review]

File: modcp.php (hunk 1 above)
Comment (Eldenroot) [reply modcp.php]: You are right :(

File: modcp.php [hunk 2]
```diff
@@ -2080,16 +2080,24 @@
...
+					update_thread_counters((int)$attachment['tid'], array("attachmentcount" => "-1"));
+				}
 			}
 		}
 
-		$plugins->run_hooks("modcp_do_modqueue_end");
+	$plugins->run_hooks("modcp_do_modqueue_end");
 
-		redirect("modcp.php?action=modqueue&type=attachments", $lang->redirect_attachmentsmoderated);
+	redirect("modcp.php?action=modqueue&type=attachments", $lang->redirect_attachmentsmoderated);
```
Comment (dvz) [inline modcp.php]: Accidental indentation changes

**Review by dvz — CHANGES_REQUESTED** [review]

File: modcp.php (hunk 2 above)
Comment (Eldenroot) [reply modcp.php]: @dvz - Idk, I see something different than Github? Maybe some sync refresh issue?

**Pull request merged**